docs/removing-documents.md

```diff
@@ -1,33 +1,15 @@
-# Withdrawing and removing documents
+# Remove / remove and redirect documents
```

Comment:
Is 'Remove' different to 'remove' 😆?

Reply:
Well no if you put it like that 🤣But as there are two different actions, either `Remove` or `Remove and redirect` this title tries to convey the content of the guidance.

Reply:
Oh yeah, I'm afraid I missed that. Would it be OK to go with 'removing'?